airflow/providers/google/cloud/operators/mlengine.py

```diff
@@ -1189,6 +1189,7 @@ def __init__(
         mode: str = 'PRODUCTION',
         labels: Optional[Dict[str, str]] = None,
         impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        hyperparameters: Optional[Dict] = None,
```

Comment:
Add parameter in class documentation and maybe a link to Google documentation can be helpful to users https://cloud.google.com/ai-platform/training/docs/using-hyperparameter-tuning

```
:param hyperparameters Make a dictionary representing your HyperparameterSpec and add it to your training input.
:type Dict
```

Reply:
Updated and pushed the code